### ISSUE TITLE
Return code should not be zero if model converts failed

### ISSUE THREAD

**Issue by del-zhenwu** [open-mmlab/mmdeploy#155]
Thanks for your bug report. We appreciate it a lot.

**Checklist**

**Describe the bug**

```
2022-02-14 15:22:22,163 - mmdeploy - INFO - torch2onnx start.
load checkpoint from local path: /opt/mmdeploy/openmmlab-ci/e2e/mmdeploy/resnet18_b16x8_cifar10_20210528-bd6371c8.pth
2022-02-14:15:22:39,root ERROR    [utils.py:41] [Errno 2] No such file or directory: '/tmp/mmclassification/demo/demo.jpg'
Traceback (most recent call last):
  File "/opt/mmdeploy/mmdeploy/utils/utils.py", line 36, in target_wrapper
    result = target(*args, **kwargs)
  File "/opt/mmdeploy/mmdeploy/apis/pytorch2onnx.py", line 88, in torch2onnx
    data, model_inputs = task_processor.create_input(img, input_shape)
  File "/opt/mmdeploy/mmdeploy/codebase/mmcls/deploy/classification.py", line 127, in create_input
    data = test_pipeline(data)
  File "/opt/conda/lib/python3.7/site-packages/mmcls/datasets/pipelines/compose.py", line 33, in __call__
    data = t(data)
  File "/opt/conda/lib/python3.7/site-packages/mmcls/datasets/pipelines/loading.py", line 48, in __call__
    img_bytes = self.file_client.get(filename)
  File "/opt/mmcv/mmcv/fileio/file_client.py", line 992, in get
    return self.client.get(filepath)
  File "/opt/mmcv/mmcv/fileio/file_client.py", line 517, in get
    with open(filepath, 'rb') as f:
FileNotFoundError: [Errno 2] No such file or directory: '/tmp/mmclassification/demo/demo.jpg'
2022-02-14 15:22:40,378 - mmdeploy - ERROR - torch2onnx failed.
root@20f1255af857:/opt/mmdeploy# echo $?
0

```
**Reproduction**

1. What command or script did you run?

```none
A placeholder for the command.
```

2. Did you make any modifications on the code or config? Did you understand what you have modified?

**Environment**

1. Please run `python tools/check_env.py` to collect necessary environment information and paste it here.
2. You may add addition that may be helpful for locating the problem, such as
    - How you installed PyTorch [e.g., pip, conda, source]
    - Other environment variables that may be related (such as `$PATH`, `$LD_LIBRARY_PATH`, `$PYTHONPATH`, etc.)

**Error traceback**

If applicable, paste the error trackback here.

```none
A placeholder for trackback.
```

**Bug fix**

If you have already identified the reason, you can provide the information here. If you are willing to create a PR to fix it, please also leave a comment here and that would be much appreciated!

**Comment by RunningLeon** [comment]
Hi, input image file `/tmp/mmclassification/demo/demo.jpg` not exists: 

> No_ such file or directory: '/tmp/mmclassification/demo/demo.jpg`

**Comment by del-zhenwu** [comment]
> /tmp/mmclassification/demo/demo.jpg

Actually I put the param intentionally, in order to verify the assertion the code returned `python tools/deploy.py ***`

**Comment by RunningLeon** [comment]
fixed in #160
